js/engine/dune

```diff
@@ -7,7 +7,9 @@
   libyaml_stubs_js
   yaml_ctypes_overrides
   semgrep.running
+  semgrep.parsing_languages
```

Comment:
I don't think we want that. The whole point of splitting semgrep/src/parsing/ in a separate semgrep/src/parsing_languages/ was to make engine.js small and not include the (huge) code of all the parsers.

Reply:
Yea I had this initially and forgot to remove it